db/migrate/20160505133601_rename_alternative_url.rb

```diff
@@ -0,0 +1,5 @@
+class RenameAlternativeUrl < ActiveRecord::Migration
+  def change
+    rename_column :unpublishings, :alternative_url, :alternative_path
```

Comment:
You cannot.  Whitehall's validations require it to be a fully qualified GOV.UK URL.